SwiftProtobuf.xcodeproj/project.pbxproj

```diff
@@ -1447,7 +1439,6 @@
 				F44F938E1DAEA76700BC5B85 /* JSONTypeAdditions.swift in Sources */,
 				AAF2EE011DFB1450007B510F /* JSONIntegerConverting.swift in Sources */,
 				F44F93801DAEA76700BC5B85 /* Google_Protobuf_Wrappers+Extensions.swift in Sources */,
-				F44F938C1DAEA76700BC5B85 /* JSONDecoder.swift in Sources */,
```

Comment:
I think I inadvertently deleted/readded that file at some point, and this is Xcode GCing the stray reference.

Reversing this out properly would be complicated, so I think I'll leave it.